ui/src/containers/App.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { withRouter, Switch, Route } from 'react-router-dom';
+import { IntlProvider, addLocaleData } from 'react-intl';
+import { ThemeProvider } from 'styled-components';
+import locale_en from 'react-intl/locale-data/en';
+import locale_fr from 'react-intl/locale-data/fr';
+import { matchPath } from 'react-router';
+import { Layout } from 'core-ui';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+import translations_en from '../translations/en';
+import translations_fr from '../translations/fr';
+
+import NodeList from './NodeList';
+import Welcome from '../components/Welcome';
+import Login from './Login';
+import PrivateRoute from './PrivateRoute';
+import { logoutAction } from '../ducks/login';
+import { toggleSidebarAction } from '../ducks/app';
+
+const messages = {
+  en: translations_en,
+  fr: translations_fr
+};
+
+addLocaleData([...locale_en, ...locale_fr]);
+
+class App extends Component {
+  // componentDidMount = () => {
+  //   fetch("/brand/theme.json")
+  //     .then(r => r.json())
+  //     .then(data => {
+  //       console.log("json", data);
+  //     });
+  // };
+
+  render() {
+    const intl = messages[this.props.language];
+    const applications = [{ label: 'Hyperdrive UI', onClick: () => {} }];
+
+    const help = [
+      {
+        label: intl.about,
+        onClick: () => {
+          this.props.history.push('/about');
+        }
+      }
+    ];
+
+    const user = {
+      name: this.props.user && this.props.user.username,
+      actions: [{ label: intl.log_out, onClick: this.props.logout }]
+    };
+
+    const sidebar = {
+      expanded: this.props.sidebar.expanded,
+      actions: [
+        {
+          label: intl.nodes,
+          icon: <i className="fas fa-server" />,
+          onClick: () => {
+            this.props.history.push('/nodes');
+          },
+          active:
+            matchPath(this.props.history.location.pathname, {
+              path: '/',
+              exact: true,
+              strict: true
+            }) ||
+            matchPath(this.props.history.location.pathname, {
+              path: '/nodes',
+              exact: true,
+              strict: true
+            })
+        }
+      ]
+    };
+
+    const navbar = {
+      onToggleClick: this.props.toggleSidebar,
+      toggleVisible: true,
+      productName: intl.product_name,
+      applications,
+      help,
+      user: this.props.user && user
+    };
+
+    const theme = {
+      brand: {
```

Comment:
I'm a bit confused why this is in here, whilst there are other configurations already specifying branding.

Reply:
We will tackle the branding part later.